File: JavaToKotlin/app/src/main/java/io/dustin/apps/board/domain/community/comment/repository/custom/CustomCommentRepository.java
package io.dustin.apps.board.domain.community.comment.repository.custom;

import io.dustin.apps.board.domain.community.comment.model.dto.CommentDto;

import java.util.List;

public interface CustomCommentRepository {

    List<CommentDto> commentListByPosting(long userId, long postingId, int size, Long nextId);

    List<CommentDto> replyListByComment(long userId, long commentId, int size, Long nextId);

}

File: apps/src/main/kotlin/io/dustin/apps/board/domain/community/comment/repository/custom/CustomCommentRepository.kt
package io.dustin.apps.board.domain.community.comment.repository.custom

import io.dustin.apps.board.domain.community.comment.model.dto.CommentDto

interface CustomCommentRepository {
    fun commentListByPosting(userId: Long, postingId: Long, recordsCount: Long, nextId: Long?): List<CommentDto>
    fun replyListByComment(userId: Long, commentId: Long, recordsCount: Long, nextId: Long?): List<CommentDto>
}
